### DISCUSSION TITLE
layers progression

### DISCUSSION THREAD

**Discussion by jekysan** [QIDITECH/QIDI_Q1_Pro#71] [Ideas]
Is it possible to inlcude how many layers have been printed in the printing progress percentage?

**Comment by CChen616** [comment]
Replace the original file with this new version of `print_stat.py` to register the `SET_PRINT_STATS_INFO` gcode macro.

Additionally, add the following line (this may vary depending on the slicer software) to the slicer's layer change gcode:

`SET_PRINT_STATS_INFO [TOTAL_LAYER=<total_layer_count>]`

This should help track the layer count, but proper display might still require support from `Fluidd` or updates to the screen.

**Reply by JdeV987** [reply]
Where do I find:  "this new version of print_stat.py"
